cmd/svcat/testdata/output/describe-plan-with-schemas.txt

```diff
@@ -0,0 +1,27 @@
+  Name:          premium                               
+  Description:   Premium plan                          
+  UUID:          cc0d7529-18e8-416d-8946-6f7456acd589  
+  Status:        Active                                
+  Free:          false                                 
+  Class:         user-provided-service                 
+
+Instances:
+No instances defined
+
+Instance Create Schema:
+properties:
```

Comment:
Would it be possible to indent the schema yaml? It isn't critical but would be easier to read. What do you think?

```console
$ svcat describe plan premium
  Name:          premium                               
  Description:   Premium plan                          
  UUID:          cc0d7529-18e8-416d-8946-6f7456acd589  
  Status:        Active                                
  Free:          false                                 
  Class:         user-provided-service                 

Instances:
No instances defined

Instance Create Schema:
  properties:
    testInstanceProperty:
      description: A test instance property.
      type: string
  required:
  - testInstanceProperty
  type: object

Binding Create Schema:
  properties:
    testBindingProperty:
      description: A test binding property.
      type: string
  required:
  - testBindingProperty
  type: object
```

Reply:
Done.